x-pack/plugins/ml/server/models/data_recognizer/modules/sample_datasets_ecommerce/ml/mean_taxful_total_price.json

```diff
@@ -0,0 +1,38 @@
+{
+  "groups": [],
+  "description": "Kibana Sample Datasets Ecommerce : detect anomalies in total sales",
+  "analysis_config" : {
+    "bucket_span": "1h",
+    "detectors": [
+      {
+        "detector_description": "Mean total sales",
+        "function": "mean",
+	"field_name": "taxful_total_price",
```

Comment:
Is the whitespace indent OK in the raw file?

Reply:
Yep, the whitespace indent is in line with the previous lines in the raw file. This must just be a GitHub rendering issue.

Reply:
one line has space characters, the other is a single tab characters.
We should only be using space characters.